templates/emo/templates.yaml

```diff
@@ -0,0 +1,60 @@
+dataset: emo
+templates:
+  2603600f-2d49-40f5-a8c6-05c9b38eab0c: !Template
+    id: 2603600f-2d49-40f5-a8c6-05c9b38eab0c
+    jinja: "Person A says something, Person B responds, and then Person A says something.\
+      \ Here's their conversation: \n\n\"{{text}}\"\n\nGiven the context and the last\
+      \ message, how would you best describe Person A's emotion - {{\"happy\"}}, {{\"\
+      sad\"}}, {{\"angry\"}}, or {{\"something else\"}}?\n\n|||\n\n{{ [\"others\"\
+      , \"happy\", \"sad\", \"angry\"] [label] }}"
+    name: persons_describe
+    reference: ''
+    task_template: true
+  4b078a4e-1a04-4401-a65e-a3d30c7d16ad: !Template
+    id: 4b078a4e-1a04-4401-a65e-a3d30c7d16ad
+    jinja: "Consider this textual dialogue of 3 messages between 2 participants who\
+      \ took turns to talk: \n\n\"{{text}}\"\"\n\nWhat would you say is the underlying\
+      \ emotion of the final message in this dialogue? {{\"happy\"}}, {{\"sad\"}},\
+      \ {{\"angry\"}}, or {{\"something else\"}}?\n\n|||\n\n{{ [\"others\", \"happy\"\
+      , \"sad\", \"angry\"] [label] }}"
+    name: final_message
+    reference: ''
+    task_template: true
+  99f6f2b9-ed9f-42f7-b0bc-249cead1a82f: !Template
+    id: 99f6f2b9-ed9f-42f7-b0bc-249cead1a82f
+    jinja: "Person A says something, Person B responds, and then Person A says something.\
+      \ Here's their conversation: \n\n\"{{text}}\"\n\nWhat emotion can be inferred\
+      \ from the context and Person A's last message? {{\"happy\"}}, {{\"sad\"}},\
+      \ {{\"angry\"}}, or  {{\"something else\"}}?\n\n|||\n\n{{ [\"others\", \"happy\"\
+      , \"sad\", \"angry\"] [label] }}"
+    name: persons_infer
+    reference: ''
+    task_template: true
+  9a87023c-176a-4031-9ef5-92a775ca9d83: !Template
+    id: 9a87023c-176a-4031-9ef5-92a775ca9d83
+    jinja: "Consider this textual dialogue of 3 messages between 2 participants who\
+      \ took turns to talk: \n\n\"{{text}}\"\n\nGiven the context, what emotion can\
+      \ you detect in the message from the person who spoke last? {{\"happy\"}}, {{\"\
+      sad\"}}, {{\"angry\"}}, or {{\"something else\"}}?\n\n|||\n\n{{ [\"others\"\
+      , \"happy\", \"sad\", \"angry\"] [label] }}"
+    name: spoke_last
+    reference: ''
+    task_template: true
+  f686cef0-6174-466d-b87d-672aaf5f9caa: !Template
+    id: f686cef0-6174-466d-b87d-672aaf5f9caa
+    jinja: '"{{text}}"
+
+
+      This was a dialogue exchange between 2 people who took turns to talk. Given
+      the context, which of the following options best describes the emotion that
+      last speaker is feeling? {{"happy"}}, {{"sad"}}. {{"angry"}}, or {{"something
+      else"}}?
```

Comment:
That doesn't correctly explain the task: you don't see it's a dialogue, there is no context/dialogue history.
Better: `This is an extract of a dialogue between two people. Which of the following options best describes the emotion the speaker is feeling?`